packages/grafbase-sdk/src/typedefs/resolver.ts

```diff
@@ -43,7 +45,22 @@ export class ResolverDefinition {
     return new CacheDefinition(this, new FieldLevelCache(params))
   }
 
+  /**
+   * Declares that this resolver requires certain fields to function correctly
+   *
+   * @param fields - The fields this resolver requires
```

Comment:
Why is this plural? It's a string param, so should it be `field: string`?

Reply:
Delimited by whitespace, with nesting.  It's basically a subset of GraphQL syntax (missing arguments & directives specifically)